heareval/predictions/runner.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env python3
+"""
+Downstream training, using embeddings as input features and learning
+predictions.
+
+TODO: Add CUDA device.
+"""
+
+from importlib import import_module
+from pathlib import Path
+from typing import Optional
+
+import click
+from tqdm import tqdm
+
+from heareval.predictions.task_predictions import task_predictions
+
+
+@click.command()
+@click.argument("module", type=str)
+@click.option(
+    "--embeddings-dir",
+    default="embeddings",
+    help="Location of task embeddings to compute predictions on",
+    type=click.Path(exists=True),
+)
+@click.option(
+    "--model",
+    default=None,
+    help="Location of model weights file",
+    type=click.Path(exists=True),
+)
+def runner(
+    module: str, embeddings_dir: str = "embeddings", model: Optional[str] = None
+) -> None:
+    embeddings_dir = Path(embeddings_dir).joinpath(module)
```

Comment:
@jorshi Where? Can you show me? Sure I'm in favor of consisteny.

Reply:
https://github.com/neuralaudio/hear-eval-kit/blob/20b7c4b4a16afd5951d9fba88289fc65c10feaf1/heareval/embeddings/task_embeddings.py#L83

Reply:
So that's after importing the module. Could do something like:
```
module = import_module(module)
embeddings_dir = Path(embeddings_dir).joinpath(module.__name__)
```

Reply:
@jorshi maybe let's just use the string everywhere? Because then we don't actually need to load code necessarily just to get the directory name.

Reply:
Sure that's a good idea. Let's do that. Let's update the task_embeddings then.

Reply:
#139